lib/shared/messages.js

```diff
@@ -67,6 +67,11 @@ module.exports = {
       'It looks like this is not a bootable image.\n\n',
       'The image does not appear to contain a partition table,',
       'and might not be recognized or bootable by your device.'
+    ].join(' ')),
+
+    largeDriveSize: _.template([
+      'It looks like you\'re trying to flash a large drive of size <%= humanDriveSize %>,',
+      'this could potentially be a backup drive containing important data.',
```

Comment:
Hmmm, not sure if we specifically want to mention "backup drive" here? Although I like the "important data" phrasing.
How about something like:
"It looks like you're trying to flash a large drive with size 120 GB, are you sure this disk doesn't contain important data? Etcher will overwrite everything."
?
I suspect @alexandrosm will want some input here :)

Reply:
We can simplify this to:

> Because of its size, this drive doesn't look like a removable drive. Are you sure you want to flash it?

Reply:
I don't think we need to mention that 64 GB is the limit that triggers the heuristic

Reply:
I think we should mention the size too, and we could also add the drive `description`. It's very helpful to the user in identifying which drive they've chosen just from that message since it covers the drive selector. What about:

> Because of its size of <%= humanDriveSize %>, <%= driveDescription %> doesn't look like a removable drive. Are you sure you want to flash it?

Reply:
I think the "removable drive" language isn't what we want either though, because somebody was complaining about accidentally overwriting their portable backup drive using Etcher - and in this case it actually _is_ a removable drive.

Reply:
What about something like:

> <%= driveDescription %> is quite large (<%= humanDriveSize %>). Please check it doesn't contain any important data before Etcher overwrites it.